### PR TITLE
First PR-- Manohar

### PR DESCRIPTION
**Pull request by manohar52**

### CODE REVIEW

File: docs/TEAM.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Micheal Aderinto, Manchester UK. Hoping this will be by first step in the world
 - [Nishank Navelkar](https://www.github.com/nishanknavelkar)
 Im from India, Basic knoweledge of Python, C, C++, HTML, CSS,  Electrical engineer, Here to learn and Contribute.
 
+- [Manohar Rajaram](https://github.com/manohar52)
+I am from India but currently pusruing my Masters in computer science in US. I am quite new to Javascript. I have some experience in HTML, CSS and PHP.
```
Comment (UnevenCoder) [inline docs/TEAM.md:29]: pursuing wrong spelling

**Review by UnevenCoder — CHANGES_REQUESTED** [review]
Added a comment .. spelling mistake 😅

File: docs/TEAM.md (hunk 1 above)
Comment (manohar52) [reply docs/TEAM.md]: Corrected the spelling.. thank you for that

**Review by nishanknavelkar — APPROVED** [review]

**Comment by nishanknavelkar** [comment]
Congrats on your first PR

**Pull request merged**